config/jobs/kubernetes/sig-arch/conformance-audit.yaml

```diff
@@ -34,7 +32,7 @@ periodics:
       - name: GINKGO_TOLERATE_FLAKES
         value: "n"
       - name: BUILD_TYPE
-        value: bazel
+        value: docker
```

Comment:
this one is not just dropped because it's using a custom script. cc @hh

Reply:
@zachmandeville can you ensure our jobs are still happy?

Reply:
this job is happy! https://k8s-testgrid.appspot.com/sig-arch-conformance#kind-conformance-audit

Reply:
and importantly building with make as intended, we are dropping bazel in k/k (notice going out tomorrow)